tools/pipelines/server-gitrest.yml

```diff
@@ -80,7 +80,7 @@ pr:
     - tools/pipelines/templates/include-vars.yml
     - tools/pipelines/templates/include-install-pnpm.yml
     - tools/pipelines/templates/include-use-node-version.yml
-    - tools/pipelines/templates/upload-dev-manifest.yml
+
```

Comment:
I'm a bit torn about this one. I know the intention is that server pipelines don't actually publish a dev manifest... but they still publish npm packages, so they include the include-publish-npm-package template, which in turn might include upload-dev-manifest. I think it might make sense to keep this just so the full tree of potential yml file dependencies is used as a trigger, in case we accidentally break something about publishing by making changes to upload-dev-manifest.

Otherwise, we should also remove this from all the non-client pipelines. But I think that'd be difficult to maintain, so I'm inclined to keep it in all pipelines that potentially publish npm packages.